docs/topics/mta-review-reports.adoc

```diff
@@ -152,6 +155,56 @@ image::3-5-expanded-issue.png[Expanded rule in the issues report]
 
 Issues are sorted into four categories by default. Information on these categories is available at ask Category.
 
+
+=== Insights

```

Comment:
I'm missing the reference to this feature being available in Tech Preview.

Reply:
incorporated.